src/MUSI6106Exec/MUSI6106Exec.cpp

```diff
@@ -96,31 +136,80 @@ int main(int argc, char* argv[])
 
         // read data (iNumOfFrames might be updated!)
         phAudioFile->readData(ppfAudioData, iNumFrames);
+        phFilterFile->process(ppfAudioData, ppfFilterData, stFileSpec.iNumChannels);
 
         cout << "\r" << "reading and writing";
+        
+        // FIR/IIR CombFilter
```

Comment:
This code should be implemented within the process method of your CCombFilter class, and not in your main function.